apt-file.mk

```diff
@@ -0,0 +1,44 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS       += apt-file
+APT-FILE_VERSION  := 3.2.2
+DEB_APT-FILE_V    ?= $(APT-FILE_VERSION)
+
+apt-file-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://salsa.debian.org/apt-team/apt-file/-/archive/debian/3.2.2/apt-file-debian-$(APT-FILE_VERSION).tar.gz 
+	$(call EXTRACT_TAR,apt-file-debian-$(APT-FILE_VERSION).tar.gz,apt-file-debian-$(APT-FILE_VERSION),apt-file)
+	mkdir -p $(BUILD_STAGE)/apt-file/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin
+
+ifneq ($(wildcard $(BUILD_WORK)/apt-file/.build_complete),)
+apt-file:
+	@echo "Using previously built apt-file."
+else
+apt-file: apt-file-setup
+	$(SED) -i '1s|.*|#!$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin/perl|g' $(BUILD_WORK)/apt-file/apt-file
+
```

Comment:
Please don't have blank lines in rules, either delete them or add a tab. They break coloring in vim.